spec/features/bucket_lists/user_edits_bucket_lists_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature 'authenticated user creates a bucket list', %(
+  As a last minute traveler or traveler who's stay is unexpectedly extended,
+  I want to edit a bucket list
+  So I can later recall the destinations without weighing down my bag or mind.
+) do
+# Acceptance Criteria
+#[x] I can change the title and description
+#[x] I must click submit to save my changes
+#[x] I can toggle between private and public
+
+
+  scenario "user successfully changes the title of a bucket list" do
+    visit new_user_session_path
+    sign_in
+    visit root_path
+    click_link 'New Bucket List!'
+    fill_in 'Title', with: 'Europe'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.